pandunia-lekse.csv

```diff
@@ -525,6 +541,7 @@ daw (dawe)|||zho:道 (dào), yue:道 (dou6), jpn:道 (dō), kor:도 (do), vie:
 dawiste||||Taoist||taoísta||||||||||||||taolainen|taoista
 dawistia||||Taoism (Daoism)||taoísmo||||||||||||||taolaisuus|taoizm
 day (dayi)|||yue:大 (daai), zho:大 (dá), vie:đại, jpn:(dai,tai), kor:대— (dae-)|big (great)||grande|||||大きい||||||||granda|iso (suuri)|duży
+day lasune||||giant onion||||лук исполинский|||||||||||||
```

Comment:
"daylasune" or "dayi lasune"? Let's use the vowel endings more consistently.

Reply:
I used an adjective since giant onions are the same genus of plant as regular onions, whereas a compound would, I think, imply a different kind of plant. So "dayi lasune" sounds good.

Reply:
OK, I get it. :)